<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewShow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget {
	background-color: rgba(32, 255, 8, 95);
}
.QWidget:hover {
	background-color: rgba(42, 206, 42, 95);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,6,1" columnstretch="1,6,1">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="topLeftGrip" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to resize
Double-click to maximize/restore</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-bottom: 3px;
margin-right: 3px;
</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="topGrip" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to resize
Double-click to maximize/restore</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-bottom: 3px;
</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="topRightGrip" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to resize
Double-click to maximize/restore</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-bottom: 3px;
margin-left: 3px;
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="leftGrip" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to resize
Double-click to maximize/restore</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-right: 3px;
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="rightGrip" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to resize
Double-click to maximize/restore</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-left: 3px;
</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="bottomLeftGrip" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to resize
Double-click to maximize/restore</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-top: 3px;
margin-right: 3px;
</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="bottomGrip" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to resize
Double-click to maximize/restore</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-top: 3px;
</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QWidget" name="bottomRightGrip" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Drag to resize
Double-click to maximize/restore</string>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-top: 3px;
margin-left: 3px;
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="centerGrip" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>90</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="startButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Start screen recording</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="quitButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>90</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
